core/domain/entities/editor/Block.php

```diff
@@ -104,21 +84,33 @@ public function editorBlockType()
     /**
      * @return string
      */
-    public function namespacedEditorBlockType()
+    public function namespacedBlockType()
     {
-        return EditorBlock::NS . $this->editor_block_type;
+        return Block::NS . $this->editor_block_type;
```

Comment:
Should we be avoiding abbreviations like `NS`?

Reply:
why? What else could it be misinterpreted as:

- Nova Scotia
- Natural Science
- Non Smoker
- Nice Shot
- Nederlandse Spoorwegen
- Naughty Simian

errr.. ok... I'll change it...

Reply:
I like Non Smoker 😆